lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1072,13 +1075,42 @@ instance NFData DecentralizationLevel
 instance Buildable DecentralizationLevel where
     build = build . unDecentralizationLevel
 
+-- | The maximum size of a serialized `TokenBundle` (`_maxValSize` in the Alonzo
+-- ledger)
+newtype TokenBundleMaxSize = TokenBundleMaxSize
+    { unTokenBundleMaxSize :: Quantity "byte" Word16 }
+    deriving (Bounded, Eq, Generic, Show)
```

Comment:
Perhaps it would be good to use the pre-existing `TxSize` type (which specifies the size in bytes of part of all of a transaction):
```suggestion
newtype TokenBundleMaxSize = TokenBundleMaxSize
    { unTokenBundleMaxSize :: TxSize }
    deriving (Bounded, Eq, Generic, Show)
 ```